vnext/Microsoft.ReactNative/Views/TouchEventHandler.cpp

```diff
@@ -537,53 +591,96 @@ const wchar_t *TouchEventHandler::GetTouchEventTypeName(TouchEventType eventType
 }
 
 bool TouchEventHandler::TagFromOriginalSource(
-    const winrt::PointerRoutedEventArgs &args,
+    winrt::Microsoft::ReactNative::ReactPointerEventArgs &args,
```

Comment:
The function previously output tag, but now also fills out the pointer args. Could we update naming, or make it clearer that this function is responsible for mutating args?

A convention is to use a pointer instead of reference for out params, with the params at the end of the function. That could help make it less ambiguous at the call-site.

Reply:
How about `PropagatePointerEventAndFindTarget`?